amazon-ecr-credential-helper.spec

```diff
@@ -139,6 +139,10 @@ install -D -m 0644 \
 rm -rf %{buildroot}
 
 %changelog
+* Thu Sep 25 2025 swpnlg <swpnlg@amazon.com> - 0.10.1-2
+- Update golang version
+- Remediate CVE-2025-4673 and CVE-2025-22874
```

Comment:
This CVE remediation line is not required, as we already called it out in the previous release